src/apps/statements/expressPresenter/putStatement.ts

```diff
@@ -13,29 +13,50 @@ import getMultipartStatements from './utils/getMultipartStatements';
 import storeStatement from './utils/storeStatement';
 import validateVersionHeader from './utils/validateHeaderVersion';
 import { validateStatementProcessingPriority } from './utils/validateStatementProcessingPriority';
+import { validateStatementBypassQueues } from './utils/validateStatementBypassQueues';
 
 export default (config: Config) => {
   return catchErrors(
     config,
     async (req: Request, res: Response): Promise<void> => {
       validateStatementProcessingPriority(req.query.priority as string | undefined);
+      validateStatementBypassQueues(req.query.bypassQueues as string[] | undefined);
```

Comment:
@lucuilearningpool , it can't be a `string[]`. All values of `req.query` whether `undefined` or a simple `string`